tests/publishing/convert-image.test.js

```diff
@@ -5,19 +5,34 @@ describe('Convert Image', () => {
   beforeEach(() => common.login(browser, 'admin', 'admin', 'local')
     .then(() => common.clickSidebarTab(browser, 'Convert Image'))
   );
-
-  // it('should navigate to Convert Image', () =>
-  //   common.clickSidebarTab(browser, 'Convert Image'));
+  afterEach(() => common.logout());
 
   it('should complain about empty source input', () => {
   	return browser.getValue('//input[@id="source"]')
   	  .then((text) => expect(text).to.equal(''))
   	  .then(() => browser.click('//button[@id="btn_convert"]'))
   	  .then(() => browser.isVisible('//div[@class="has-error"]'))
   });
+  
   it('should complain about invalid source input', () => {
     return browser.setValue('//input[@id="source"]', 'invalid input')
       .then(() => browser.click('//button[@id="btn_convert"]'))
       .then(() => browser.isVisible('//div[@class="has-error"]'))
-  });  
+  });
+
+  it('should clear old source error messages', () => {
+    return browser.setValue('//input[@id="source"]', 'invalid input')
+      .then(() => browser.click('//button[@id="btn_convert"]'))
+      .then(() => browser.setValue('//input[@id="source"]', ''))
+      .then(() => browser.click('//button[@id="btn_convert"]'))
+      .then(() => browser.waitForExist('//input[@id="source"]/..//i[position()=2]',500,true))
+  });
+
+  it('should clear old output error messages', () => {
+    return browser.setValue('//input[@id="output"]', 'invalid input')
+      .then(() => browser.click('//button[@id="btn_convert"]'))
+      .then(() => browser.setValue('//input[@id="output"]', ''))
+      .then(() => browser.click('//button[@id="btn_convert"]'))
+      .then(() => browser.waitForExist('//input[@id="output"]/..//i[position()=2]',500,true))
+  });      
 });
```

Comment:
If the conversion is done on a large file, the current implementation, will eventually stop the spinner, and alert the user that the conversion is still running (this is due to the shitty way it is implemented, that causes the underlying Nginx session to timeout) - not sure we want to test that, just bringing it up...

* noticed that not conversion test actually takes place